pkg/core/waitfor/waitfor.go

```diff
@@ -53,7 +53,7 @@ func (Condition) LoadTestFinished(event watch.Event) (bool, error) {
 
 // Resource waits until a kubernetes resources to match a condition
 func Resource(obj watch.Interface, condFunc watchtools.ConditionFunc) (*watch.Event, error) {
-	ctx, cancel := context.WithTimeout(context.Background(), kubernetes.KubeTimeout)
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
```

Comment:
Hard coding this could cause a similar issue that you are trying to fix with `KUBE_CLIENT_TIMEOUT` maybe add a to do to introduce a `KUBE_WAIT_FOR_TIMEOUT` or something?

Reply:
Added in ae26927